test/test/recovery/RecoveryTests.java

```diff
@@ -17,7 +17,7 @@ public class RecoveryTests {
     public void stringBuilder(TestProcess p) throws Exception {
         Output out = p.profile("-d 3 -e cpu -o collapsed");
 
-        Assert.isGreater(out.ratio("StringBuilder.delete;"), 0.9);
+        Assert.isGreater(out.ratio("StringBuilder.delete;"), 0.8);
```

Comment:
Can you please provide profiler output where the ratio is less than 0.9?
It should be in `build/test/logs`.

Reply:
https://github.com/openorclose/async-profiler/actions/runs/11324932513/artifacts/2052406069

https://github.com/openorclose/async-profiler/actions/runs/11324932513/job/31491243794#step:4:88

Although this failure is for the next line checking for `arraycopy`, so I'll reduce that too

output:
```
test/recovery/StringBuilderTest.main;I2C/C2I adapters 1
thread_start;_pthread_start;thread_native_entry;Thread::call_run;WatcherThread::run;WatcherThread::sleep;Monitor::wait;Monitor::IWait;os::PlatformEvent::park;__psynch_cvwait 1
test/recovery/StringBuilderTest.main;java/lang/Thread.currentThread;JVM_CurrentThread 1
test/recovery/StringBuilderTest.main;java/lang/Thread.currentThread;ThreadStateTransition::trans_from_native 1
test/recovery/StringBuilderTest.main;java/lang/Thread.currentThread;JVM_CurrentThread 2
test/recovery/StringBuilderTest.main;java/lang/StringBuilder.delete;java/lang/AbstractStringBuilder.delete;java/lang/AbstractStringBuilder.shift 1
test/recovery/StringBuilderTest.main;java/lang/StringBuilder.delete;java/lang/AbstractStringBuilder.delete;java/lang/AbstractStringBuilder.shift;jbyte_disjoint_arraycopy 116
thread_start;_pthread_start;thread_native_entry;Thread::call_run;JavaThread::run;JavaThread::thread_main_inner;CompileBroker::compiler_thread_loop;CompileBroker::invoke_compiler_on_method;C2Compiler::compile_method;Compile::Compile;Compile::Code_Gen;PhaseCFG::PhaseCFG;PhaseCFG::build_cfg 1
test/recovery/StringBuilderTest.main;java/lang/Thread.isAlive;JVM_IsThreadAlive;JavaThread::is_Java_thread 2
test/recovery/StringBuilderTest.main;java/lang/Thread.currentThread 1
test/recovery/StringBuilderTest.main;java/lang/StringBuilder.delete;java/lang/AbstractStringBuilder.delete;java/lang/AbstractStringBuilder.shift 1
test/recovery/StringBuilderTest.main;java/lang/StringBuilder.append;java/lang/AbstractStringBuilder.append 1
test/recovery/StringBuilderTest.main;java/lang/StringBuilder.delete;java/lang/AbstractStringBuilder.delete;java/lang/AbstractStringBuilder.shift;jbyte_disjoint_arraycopy 41
test/recovery/StringBuilderTest.main;java/lang/Thread.isAlive;JVM_IsThreadAlive;ThreadStateTransition::trans_from_native;InterfaceSupport::serialize_thread_state_internal 1
test/recovery/StringBuilderTest.main;java/lang/StringBuilder.delete;java/lang/AbstractStringBuilder.delete;java/lang/AbstractStringBuilder.shift;jbyte_disjoint_arraycopy 1
thread_start;_pthread_start;thread_native_entry;Thread::call_run;ConcurrentGCThread::run;G1YoungRemSetSamplingThread::run_service;G1YoungRemSetSamplingThread::sleep_before_next_cycle;Monitor::wait;Monitor::IWait;os::PlatformEvent::park;__psynch_cvwait 1
test/recovery/StringBuilderTest.main;java/lang/Thread.currentThread;JVM_CurrentThread;JavaThread::is_Java_thread 1
test/recovery/StringBuilderTest.main;java/lang/Thread.currentThread 1
test/recovery/StringBuilderTest.main;java/lang/Thread.isAlive;JVM_IsThreadAlive 1
test/recovery/StringBuilderTest.main;java/lang/StringBuilder.delete;java/lang/AbstractStringBuilder.delete;java/lang/AbstractStringBuilder.shift;jbyte_disjoint_arraycopy 1
test/recovery/StringBuilderTest.main;java/lang/Thread.currentThread;JVM_CurrentThread;JNIHandleBlock::allocate_handle 1

```

Reply:
OK, so it seems that native method call overhead is higher on macOS. Otherwise tests results make sense.
It's fine to lower threshold in the test to 0.8 then.